iscsi/iscsi.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2017 The Prometheus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iscsi
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/prometheus/procfs/internal/fs"
+)
+
+// ConfigFsPath is static configfs mount point
+const ConfigFsPath = "/kernel/config"
+
+// TargetPath is static sys path for iscsi target
+const TargetPath = "/target/iscsi"
+
+// TargetCore static sys path for backstore
+const TargetCore = "/target/core"
```

Comment:
This module was originally split from https://github.com/AvengerMoJo/node_exporter/tree/lio_rework
Which also need to rework now since I have not touch it for a year as well.

Reply:
That's fine, if you expect it to be used from outside the package, you can just leave it.  But maybe remove the first slash to make it clear that it's a relative path and not absolute.